zwave_js_server/model/node.py

```diff
@@ -234,6 +240,25 @@ def interview_attempts(self) -> Optional[int]:
         """Return the interview_attempts."""
         return self.data.get("interviewAttempts")
 
+    def get_command_class_values(
+        self, command_class: CommandClass, endpoint: int = None
+    ) -> Optional[Dict[str, Value]]:
```

Comment:
We always return a dict.
```suggestion
    ) -> Dict[str, Value]:
```